geodocker-geoserver/Dockerfile.template

```diff
@@ -1,22 +1,28 @@
-FROM quay.io/geomesa/base:__TAG__
+FROM centos:7
+ARG GEOMESA_VERSION
+RUN curl -sSL \
+    https://repo.locationtech.org/content/repositories/geomesa-releases/org/locationtech/geomesa/geomesa-accumulo-dist_2.11/${GEOMESA_VERSION}/geomesa-accumulo-dist_2.11-${GEOMESA_VERSION}-bin.tar.gz \
+  | tar -zx
 
-ARG TAG
+FROM centos:7
```

Comment:
I don't Docker much; does this 'FROM' directive need to be repeated here?

Reply:
should only be at top

Reply:
These multiple FROM statements are used for multi-stage builds in Docker (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds).
I did this to help reduce the number of layers in the final image and therefore reduce the final image size. It used to be ~2GB, but with multi-stage builds it's down to ~1GB

Reply:
whaaat version 17.05 man I'm old and behind. This is cool.